BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Jimmy/JimmyViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  JimmyViewController.swift
+//  BackToBasicsUIKit
+//
+//  Created by Jimmy Gutierrez on 5/23/19.
+//  Copyright © 2019 SwiftMiami. All rights reserved.
+//
+
+import UIKit
+
+class JimmyViewController: UIViewController, canRecieve {
+
+    @IBOutlet var label: UILabel!
+    @IBOutlet var textField: UITextField!
+    
+    override func viewDidLoad() {
+
+
+    }
+    
+    
+    @IBAction func buttonPressed(_ sender: Any) {
+        performSegue(withIdentifier: "sendDataForwards", sender: self)
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "sendDataForwards" {
+            
+            let secondVC = segue.destination as! JimmySecondViewController
+            
+            secondVC.data = textField.text!
```

Comment:
The guard above should also check for the text field contents, rather than force unwrapping.

Reply:
If `JimmySecondViewController` requires a string to be used, then it should initialized with it.